app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::CommentsController < ApplicationController
+  protect_from_forgery with: :null_session
+  def new
+    @comment = Comment.new
+    respond_to do |format|
+      format.json { render json: @comment }
+    end
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = current_user.comments.new(post_id: @post.id, author_id: current_user.id, Text: comment_params)
+    @comment.post_id = @post.id
+    return unless
+    @comment.save
+
+    respond_to do |format|
+      format.json { render json: @comment, status: 'Created' }
+    end
+  end
+
+  def comment_params
+    params.require(:comment).permit(:text)[:text]
+  end
+end
```

Comment:
- [ ] Please create an API endpoint to list all comments for a user's posts requested in the project requirements.